# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/NeckoChild.cpp
# Commit: d4004105a04a
# Full Hash: d4004105a04ad9a473930acb9c6a05d90cd9f028
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-02-23 04:13:22
# Regressor Bug: 1521879
# File Overlap Count: 1
# Description:
#   Bug 1521879 - Part 1: IPC-based MediaTransport implementation r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D17273
# ==============================================================================

diff -r baecf588657c -r d4004105a04a netwerk/ipc/NeckoChild.cpp
--- a/netwerk/ipc/NeckoChild.cpp	Thu Feb 21 23:16:40 2019 +0000
+++ b/netwerk/ipc/NeckoChild.cpp	Thu Feb 21 20:24:54 2019 +0000
@@ -24,6 +24,7 @@
 #include "mozilla/dom/network/UDPSocketChild.h"
 #include "mozilla/net/AltDataOutputStreamChild.h"
 #include "mozilla/net/TrackingDummyChannelChild.h"
+#include "mozilla/net/SocketProcessBridgeChild.h"
 #ifdef MOZ_WEBRTC
 #  include "mozilla/net/StunAddrsRequestChild.h"
 #  include "mozilla/net/WebrtcProxyChannelChild.h"
@@ -67,6 +68,7 @@
     }
     gNeckoChild = cpc->SendPNeckoConstructor();
     NS_ASSERTION(gNeckoChild, "PNecko Protocol init failed!");
+    SocketProcessBridgeChild::GetSocketProcessBridge();
   }
 }
 